backend/coins/eth/account.go

```diff
@@ -479,13 +479,17 @@ func (account *Account) Notifier() accounts.Notifier {
 
 // Transactions implements accounts.Interface.
 func (account *Account) Transactions() (accounts.OrderedTransactions, error) {
-	account.Synchronizer.WaitSynchronized()
+	if !account.Synced() || account.Offline() != nil {
+		return nil, accounts.ErrSyncInProgress
```

Comment:
it's not in progress if Offline()? :thinking:

Reply:
Done